### DISCUSSION TITLE
App show no friends

### DISCUSSION THREAD

**Discussion by Havouza15** [matiasdelellis/facerecognition#724] [Q&A]
Ubuntu 22.04
NC 28
PHP 8.1

I have installed Face recognition by the book and it seems that it works. I have about 300 images and I have runt the initial scan using occ.
It say Analyzing images - 272 images detected - 29 images in queue - Ends approximately om 2 minuter. That is 8 hours ago. And it still say the same
I followed the scan and it found 12 faces,
But on the users page it say be patient, no friend found.
Is this really correct?

**Comment by matiasdelellis** [comment]
Hi @Havouza15 
Unless you have used the --defer-cluster option you must run it again to generate the clusters.
